sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/StateStoreCoordinator.scala

```diff
@@ -66,9 +86,9 @@ object StateStoreCoordinatorRef extends Logging {
   /**
    * Create a reference to a [[StateStoreCoordinator]]
    */
-  def forDriver(env: SparkEnv): StateStoreCoordinatorRef = synchronized {
+  def forDriver(env: SparkEnv, conf: SQLConf): StateStoreCoordinatorRef = synchronized {
     try {
-      val coordinator = new StateStoreCoordinator(env.rpcEnv)
+      val coordinator = new StateStoreCoordinator(env.rpcEnv, conf)
```

Comment:
you are passing in current session conf, but if i remember correctly we only have one coordinator across all sessions. I don't think we create a coordinator per session. Can you confirm?

Reply:
Hmm yeah just confirmed but we build the coordinator on a session basis. Do you have any good ways on how we can set these coordinator configurations for the log thresholds? I previously tried `SQLConf.get` but this doesn't seem to update properly during testing (is there some way to query for a specific SQLConf?)

The other alternative is to set it in SparkSession config, but I don't know if that's the right approach either.